frontend/src/components/pages/admin/AdminDashboard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Box, VStack } from "@chakra-ui/react";
+import { Switch, Route, Redirect } from "react-router-dom";
+
+import Navbar from "../../common/Navbar";
+import Page from "../../../types/PageTypes";
+import * as Routes from "../../../constants/Routes";
+import UsersPage from "./UsersPage";
+import CreateQuestionPage from "../../assessment-creation/CreateQuestionPage";
+import PrivateRoute from "../../auth/PrivateRoute";
+import NotFound from "../NotFound";
+
+const pages: Page[] = [
+  { title: "Assessments", url: Routes.ASSESSMENTS },
+  { title: "Database", url: Routes.USER_DATABASE },
+];
```

Comment:
We should probably have some constant object with all the pages urls and components and then map them to the navigation bar and routes... but maybe we can do that when we have more Routes. LOL